app/views/users/new.erb

```diff
@@ -0,0 +1,26 @@
+<form action='/users' method='post' class="form-signin">
+  <label for='name' class='sr-only'>Name:</label>
+  <input placeholder="Name" type='text' name='name' id='name' class="form-control" required>
+
+  <label for='username' class='sr-only'>Username:</label>
+  <input placeholder="Username" type='text' name='username' id='username' class="form-control" required>
+
+  <label for='email' class='sr-only'></label>
+  <input placeholder="Email address" type='email' name='email'  id='email' class="form-control" value="<%=@email%>" required>
+
+  <label for='password' class='sr-only'></label>
+  <input placeholder="Password" type='text' name='password' id='password' class="form-control">
+
+  <label for='confirm_password' class='sr-only'></label>
+  <input placeholder="Confirm password" type='text' name='confirm_password' id='confirm_password' class="form-control" >
+
+  <input class="btn btn-lg btn-primary btn-block" type='submit' value='Submit'>
+</form>
+
+<% if flash[:errors] %>
```

Comment:
Is there a way of styling flash errors? Had a go in the core stylesheet file - but it didn't work. Seems that maybe you need to make a separate stylesheet for flash errors but again, didn't have time to properly look into it.